api/src/paths/project/{projectId}/survey/{surveyId}/summary/submission/get.ts

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../../../../../../constants/roles';
+import { getDBConnection } from '../../../../../../../database/db';
+import { HTTP400 } from '../../../../../../../errors/CustomError';
+import { getLatestSurveySummarySubmissionSQL } from '../../../../../../../queries/survey/survey-summary-queries';
+import { getLogger } from '../../../../../../../utils/logger';
+
+const defaultLog = getLogger('/api/project/{projectId}/survey/{surveyId}/summary/submission/get');
+
+export const GET: Operation = [getSurveySummarySubmission()];
+
+GET.apiDoc = {
+  description: 'Fetches an summary occurrence submission for a survey.',
+  tags: ['summary_submission'],
+  security: [
+    {
+      Bearer: [SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.PROJECT_ADMIN]
+    }
+  ],
+  parameters: [
+    {
+      in: 'path',
+      name: 'projectId',
+      schema: {
+        type: 'number'
+      },
+      required: true
+    },
+    {
+      in: 'path',
+      name: 'surveyId',
+      schema: {
+        type: 'number'
+      },
+      required: true
+    }
+  ],
+  responses: {
+    200: {
+      description: 'Summary submission get response.',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            properties: {
+              id: {
+                type: 'number'
+              },
+              fileName: {
+                description: 'The file name of the submission',
+                type: 'string'
+              },
+              status: {
```

Comment:
Status and messages is not part of the response so can be removed from here

Reply:
Good catch!  Yeah .. messasges will be there soon ... but not sure in what context yet.